front/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@auth0/nextjs-auth0": "^3.5.0",
     "@dust-tt/client": "file:../sdks/js",
-    "@dust-tt/sparkle": "^0.2.284",
+    "@dust-tt/sparkle": "^0.2.287-rc-2",
```

Comment:
Make sure to use the right version!

Reply:
Ofc sir, mainly releasing candidates for the CI to pass 😇